pages/en/lb2/Deploy-for-GDPR-readiness.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Deploy for GDPR Readiness"
+layout: page
+keywords: LoopBack, GDPR
+tags: gdpr
+lang: en
+sidebar: lb2_sidebar
+permalink: /doc/en/lb2/Deploy-for-GDPR-readiness.html
+summary: LoopBack considerations for GDPR readiness.
+---
+
+# LoopBack considerations for GDPR readiness
+ 
+___________________________________________________________________________________________________
+### For PID(s): UT:30AH1
+
+## Notice:
+
+This document is intended to help you in your preparations for GDPR readiness. It provides information about features of LoopBack that you can configure, and aspects of the product’s use, that you should consider to help your organization with GDPR readiness. This information is not an exhaustive list, due to the many ways that clients can choose and configure features, and the large variety of ways that the product can be used in itself and with third-party applications and systems.
```

Comment:
I don't think we need any of the commas in this paragraph.

Reply:
will ignore because it's part of the template